src/Psalm/Internal/CallMap.php

```diff
@@ -9954,7 +9954,8 @@
 'preg_filter' => ['null|string|string[]', 'regex'=>'mixed', 'replace'=>'mixed', 'subject'=>'mixed', 'limit='=>'int', '&w_count='=>'int'],
 'preg_grep' => ['array', 'regex'=>'string', 'input'=>'array', 'flags='=>'int'],
 'preg_last_error' => ['int'],
-'preg_match' => ['int|false', 'pattern'=>'string', 'subject'=>'string', '&w_subpatterns='=>'string[]', 'flags='=>'int', 'offset='=>'int'],
+'preg_match' => ['int|false', 'pattern'=>'string', 'subject'=>'string', '&w_subpatterns='=>'string[]', 'flags='=>'0|', 'offset='=>'int'],
```

Comment:
Typing flags as `0` doesn't work, but `0|` does. Is this a bug in `Type::parseTokens()`?

Reply:
Yes, at some point I felt that declaring a single value type was useless, but I now see its value in some situations (e.g. here)